shadow-cljs.edn

```diff
@@ -112,7 +112,9 @@
                        ;; instrumenting functions.
                        :static-fns         false
                        :infer-externs      true
-                       :reader-features    #{:mobile}}
+                       :reader-features    #{:mobile
```

Comment:
Maybe this part can benefit from some explanatory comment, wdyt?

Reply:
Yup yup I’ll add a comment 🙏